myproject/app/api/user.py

```diff
@@ -116,71 +116,38 @@ def change_password():
     db.session.commit()
     return success_response('Password changed successfully', 200)
 
+
 @blueprint.route('/forget', methods=['POST'])
 def forget_password():
     data = request.json
     email = data.get('email')
-    user = rechange_link(email)
+ 
+    User = user.query.filter_by(email=email).first()
     if not user:
         return e_response('404')
+    expire_time = int(time.time()) + 60
+    password_reset_tokens[email] = expire_time
     reset_token = base64.b64encode(email.encode('utf-8')).decode('utf-8')
-    password_reset_tokens[reset_token] = {
-        'email': email,
-        'expiry_time': time.time() + 60 
-    }
-    reset_link = f'http://127.0.0.1:5000/reset/{reset_token}'
+    reset_link = basesit.REST_LINK + reset_token
     send_reset_password_email(email, reset_link)
     return success_response('Reset password link sent to your email', 200)
 
 
-def send_reset_password_email(user_email, reset_link):
-    msg = Message('Password Reset Link', sender='your_email@gmail.com', recipients=[user_email])
-    msg.body = f'Hello,\n\nYour reset link is: {reset_link}'
-    mail.send(msg)
 @blueprint.route('/reset/<reset_token>', methods=['POST'])
 def reset(reset_token):
     data = request.json
     new_password = data.get('new_password')
     confirm_password = data.get('confirm_password')
     if new_password != confirm_password:
         return e_response('409')
-    token_data = password_reset_tokens.get(reset_token)
-    if not token_data or time.time() > token_data['expiry_time']:
+    email = base64.b64decode(reset_token.encode('utf-8')).decode('utf-8')
+    if email not in password_reset_tokens or time.time() > password_reset_tokens[email]:
```

Comment:
link hould only  be used by user once but here we can use multiple times with in 1 hour